indico/modules/rb_new/controllers/backend/bookings.py

```diff
@@ -133,10 +133,10 @@ def _process(self, args):
 
         serialized_occurrences = serialize_occurrences(group_by_occurrence_date(resv.occurrences.all()))
         if is_prebooking:
-            data = {'pre_bookings': serialized_occurrences}
+            data = {'pre_booking_occurrences': serialized_occurrences}
         else:
-            data = {'bookings': serialized_occurrences}
-        return jsonify(room_id=room.id, **data)
+            data = {'booking_occurrences': serialized_occurrences}
```

Comment:
the reducer doesn't know about this rename so new bookings don't show up on the calendar now

Reply:
This is actually not intentional, now that I see it. I forgot to undo these changes.